### PR TITLE
feature-and-test: add endless pagination for tweets list api with tests

### PR DESCRIPTION
**Pull request by W-Shih**
- feature-and-test: add endless pagination for tweets list api with tests
- style: Update pylint score

### CODE REVIEW

File: tweets/api/pagination.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+from rest_framework.request import Request
+from rest_framework.response import Response
+from rest_framework.utils.urls import replace_query_param
+
+
+class TweetPagination(BasePagination):
+    page_size = 20
+
+    def __init__(self):
+        # super(TweetPagination, self).__init__()
```
Comment (wen-wayne-shih) [inline tweets/api/pagination.py:27]: clean up un-necessary code

File: tweets/api/pagination.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        super().__init__()
+        self.request = None
+        self.has_next = False
+
+    def paginate_queryset(self, queryset: QuerySet, request: Request, view=None):
+        self.request = request
+        # if 'page_size' in request.query_params:
+        #     self.page_size = int(request.query_params['page_size'])
```
Comment (wen-wayne-shih) [inline tweets/api/pagination.py:35]: clean up un-necessary code

**Review by wen-wayne-shih — CHANGES_REQUESTED** [review]
clean up un-necessary code

File: tweets/api/pagination.py (hunk 1 above)
Comment (W-Shih) [reply tweets/api/pagination.py]: Resolved as suggestion

File: tweets/api/pagination.py (hunk 2 above)
Comment (W-Shih) [reply tweets/api/pagination.py]: Resolved as suggestion

**Review by wen-wayne-shih — APPROVED** [review]
squash before merge

**Pull request merged**